### PR TITLE
Cleaned up a few things and is now using data from the api

### PR DESCRIPTION
**Pull request by zman811**
Started using the api to get data. Right now it only is using a hardcode id but that will change from the overview

### CODE REVIEW

**Comment by zman811** [comment]
ah the tests are failing cuz its going to the API

**Comment by zman811** [comment]
this is ready for review now

**Review by slargman — APPROVED** [review]
Looks great overall. I had a couple of questions because I wasn't sure about the outcome or the desired result in couple of cases, but it's ready for merge imo.

File: client/src/components/Reviews/Reviews.jsx [hunk 1]
```diff
@@ -15,15 +19,39 @@ const Left = styled.div`
...
 // TODO meta data req will happen here
-function Reviews() {
+function Reviews({ productId }) {
+  const [reviewsMetadata, setReviewsMetaData] = useState('');
+  const [productName, setProductName] = useState('');
+  useEffect(() => {
+    axios.get(`/reviews/meta?product_id=${productId}`)
+      .then((res) => { setReviewsMetaData(res.data); })
+      .catch((err) => console.log(err));
+    axios.get(`/products/${productId}`)
```
[the remark this answers is not in the document]
Comment (zman811) [reply client/src/components/Reviews/Reviews.jsx:39]: sounds good

File: client/src/components/Reviews/components/ReviewList.jsx [hunk 2]
```diff
@@ -2,56 +2,58 @@ import React, { useEffect, useState } from 'react';
...
 `;
 const StyledSelect = styled.select`
   border: none;
 `;
 
-function ReviewList() {
+function ReviewList({reviewsMetadata}) {
+  const productId = reviewsMetadata.product_id;
   const [currentSort, setCurrentSort] = useState('relevant');
   const [toggleModal, setToggleModal] = useState(false);
```
[the remark this answers is not in the document]
Comment (zman811) [reply client/src/components/Reviews/components/ReviewList.jsx:33]: yeah i was actually thinking about that changing that anyways

File: client/src/components/Reviews/components/ReviewList.jsx [hunk 3]
```diff
@@ -2,56 +2,58 @@ import React, { useEffect, useState } from 'react';
...
     setCurrentSort(e.target.value);
   };
   const handleModalToggle = () => {
     setToggleModal(!toggleModal);
-    // TODO this will pull up the review modal window to allow for the user to add a review
   };
   const handleMoreReviews = () => {
-    // TODO make a request to the api to get more/will change which reviews are being displayed
+    setPage(page + 1);
+    axios.get(`/reviews?product_id=${productId}&sort="${currentSort}"&count=2&page=${page}`)
```
[the remark this answers is not in the document]
Comment (zman811) [reply client/src/components/Reviews/components/ReviewList.jsx:51]: I updated this in a later req so disregard that

File: client/src/components/Reviews/components/ReviewList.jsx [hunk 4]
```diff
@@ -2,56 +2,58 @@ import React, { useEffect, useState } from 'react';
...
+      .then((res) => {
+        res.data.results = [...res.data.results, ...reviews.results];
```
[the remark this answers is not in the document]
Comment (zman811) [reply client/src/components/Reviews/components/ReviewList.jsx:53]: same thing here, i changed the sort to work in a later pr

**Pull request merged**